int32 nodeID
bool status
